site/src/examples/card/Accents.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+import {
+  H3,
+  Text,
+  StackLayout,
+  Card,
+  CardProps,
+  RadioButtonGroup,
+  RadioButton,
+  Link,
+} from "@salt-ds/core";
+
+export const Accents = () => {
+  const [placement, setPlacement] = useState<CardProps["accent"]>("top");
+
+  return (
+    <StackLayout style={{ width: "266px" }} align="center">
+      <Card accent={placement} hoverable>
+        <StackLayout gap={1} align="start">
+          <H3>Sustainable investing products</H3>
+          <Text>
+            We have a commitment to provide a wide range of investment solutions
+            to enable you to align your financial goals to your values.
+          </Text>
+        </StackLayout>
+      </Card>
+      <RadioButtonGroup
+        value={placement}
+        onChange={(e) => setPlacement(e.target.value as CardProps["accent"])}
```

Comment:
Nit - we should follow our own standards and not shorten names i.e. this should be `event`